recipe/meta.yaml

```diff
@@ -1,49 +1,143 @@
 {% set name = "xgboost" %}
 {% set version = "0.80" %}
-{% set sha256 = "ec990238943988e73734ad483cf6c53b81aec9745811c7b6c5cbb189ef5e328b" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  fn: {{ name }}-{{ version }}.tar.gz
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: {{ sha256 }}
+  git_url: https://github.com/dmlc/xgboost
+  # corresponds to 0.80 tag
+  git_tag: 96826a35155d42f78cd4253b3d134a208dbbaccb
```

Comment:
Am curious why this was changed to clone the git repo. Thoughts?

Reply:
It is needed for the way the build is set up to combine the R and python recipes.

Reply:
Ah it also uses submodules. Makes sense. Thanks for the help.

Reply:
Yeah the submodules too.